next.config.js

```diff
@@ -17,7 +17,7 @@ const nextConfig = {
     : undefined,
   staticPageGenerationTimeout: 180, //arbitrary; 60 is default but it's too small
   experimental: {
-    largePageDataBytes: 550, // 128kb by default
+    largePageDataBytes: 512 * 1000, // 512kb, is 128kb by default
```

Comment:
@ryguyk This should reduce some of the page data spam while still providing relevant alerts that a page is huge.